### PR TITLE
SCP-106 tweaks & the femur breaker

### PR DESCRIPTION
**Pull request by kachnov**

### CODE REVIEW

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]
Phase through containment airlock wasn't an issue.

**Comment by JakeDaBoss** [comment]
Merged anyway, we can change it later on.

File: code/modules/SCP/SCPs/SCP-106.dm [hunk 1]
```diff
@@ -321,3 +314,24 @@ GLOBAL_LIST_EMPTY(scp106s)
...
+			visible_message("<span class = 'danger'>[target] puts [user] into the femur breaker.</span>")
+			var/mob/living/carbon/human/H = target
+			H.forceMove(get_turf(src))
+			for (var/obj/item/organ/external/leg/L in H.organs)
+				if (!(L.status & BROKEN))
+					L.fracture()
+					spawn (10 SECONDS)
+						for (var/scp106 in GLOB.scp106s)
+							var/atom/movable/A = scp106 
+							A.forceMove(pick(GLOB.scp106_spawnpoints))
```
Comment (ghost) [inline code/modules/SCP/SCPs/SCP-106.dm:336]: could you nest any fucking deeper jesus christ I can see the earth's core from this depth

Comment (kachnov) [reply code/modules/SCP/SCPs/SCP-106.dm:336]: yes

**Pull request merged**